alvr/xtask/src/dependencies.rs

```diff
@@ -101,11 +120,12 @@ pub fn prepare_linux_deps(nvenc_flag: bool) {
     sh.remove_path(&deps_path).ok();
     sh.create_dir(&deps_path).unwrap();
 
-    build_x264_linux(&deps_path);
-    build_ffmpeg_linux(nvenc_flag, &deps_path);
+    prepare_openvr_sdk();
+    prepare_x264_linux(&deps_path);
+    prepare_ffmpeg_linux(nvenc_flag, &deps_path);
 }
 
-pub fn build_x264_linux(deps_path: &Path) {
+pub fn prepare_x264_linux(deps_path: &Path) {
```

Comment:
You don't need to rename, "build" would still be correct, same for the other functions

Reply:
I made that because it makes it consistent with the rest which also builds if you would like it to be the other way around I can do that.

Reply:
I'm fine either way, I don't have strong opinions